lib/theme.json

```diff
@@ -391,6 +443,16 @@
 					"width": "0"
 				}
 			},
+			"textInput": {
+				"border": {
+					"width": "1px",
+					"style": "solid",
+					"color": "#949494"
```

Comment:
not all blocks defined a color, but some did, I went ahead and made the decision to align them all on a default ($gray-600)